src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -42,7 +42,7 @@ const AudioInputVFControl: React.FC<Props> = ({
   const meetingManager = useMeetingManager();
   const [isLoading, setIsLoading] = useState(false);
   // When the user click on Amazon Voice Focus option, the state will change.
-  const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
+  const [isVoiceFocusOn, setIsVoiceFocusOn] = useState<boolean | undefined>(undefined);
```

Comment:
Earlier this was `false` now `undefined`, checking if you tested cause earlier it meant that voice focus will be off by default but now its undefined. Suggest to keep this as `false`.

If you do this for logging, below code should also work and give you initial as well as updated states right?
```
useEffect(() => {
    console.info(`Amazon Voice Focus state isVoiceFocusOn: ${isVoiceFocusOn}`);
  }, [isVoiceFocusOn]);
```

Hooks are executed in the order, so I think initial state should also get printed here.

Reply:
It makes sense to keep this as false default - we want to reduce the usage of `null` and replace those with `undefined` (if it makes sense) - this doesnt apply to booleans.

Reply:
Also - is it possible to rename this to `isVoiceFocusCheckboxOn`?

Reply:
The reason to set the default value as `undefined` is that I don't want to show the initial state in the browser, since it would be `false` every time at the beginning. So I would like to avoid this unnecessary log.  But seems like it is okay to print `false` as initial state every time when user joins the meeting?

`isVoiceFocusCheckboxOn` -> it is clearer, will update the name